bin/codemods/run.js

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env node
+
+/**
+ * External dependencies
+ */
+const fs = require( 'fs' );
+const path = require( 'path' );
+const child_process = require( 'child_process' );
+const glob = require( 'glob' );
+
+/**
+ * Internal dependencies
+ */
+const config = require( './src/config' );
+
+function getLocalCodemodFileNames() {
+	// Returns all JS files in bin/codemods/src folder, except for config and helpers.
+	return glob.sync( 'bin/codemods/src/*.js' )
+		.map( name => path.basename( name, '.js' ) ) // strip path and extension from filename
```

Comment:
Instead of using `path.basename`, we might achieve the same effect by playing with `glob`'s options `cwd` and `root` :wink: